helmfile/values/harbor.yaml.gotmpl

```
@@ -106,3 +114,41 @@ notary:
 database:
   internal:
     password: {{ .Values.harbor.databasePassword }}
+    resources: {{- toYaml .Values.harbor.database.resources | nindent 6 }}
+    nodeSelector: {{- toYaml .Values.harbor.nodeSelector | nindent 6 }}
+    affinity: {{- toYaml .Values.harbor.affinity | nindent 6 }}
+    tolerations: {{- toYaml .Values.harbor.tolerations | nindent 6 }}
```

Feel free to allign these if you want :)
```suggestion
    resources:    {{- toYaml .Values.harbor.database.resources | nindent 6 }}
    nodeSelector: {{- toYaml .Values.harbor.nodeSelector | nindent 6 }}
    affinity:     {{- toYaml .Values.harbor.affinity | nindent 6 }}
    tolerations:  {{- toYaml .Values.harbor.tolerations | nindent 6 }}
```